data/GalaxyStellarMassGasMetallicity/conversion/convertStrom2022.py

```diff
@@ -0,0 +1,80 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+# Meta-data
+comment = "The median stellar mass is 1e10 Msun."
+citation = "Strom et al. (2022)"
+bibcode = "2022ApJ...925..116S"
+plot_as = "line"
+redshift = np.array([2.0, 2.7])
+
+M_star = unyt.unyt_array(np.logspace(9.0, 11.3, 100), units="Msun")
+
+models = {
+    "OH": {"alpha": (8.35, 0.02), "beta": (0.14, 0.05), "label": "12+log10(O/H)"},
+    "NH": {"alpha": (7.07, 0.03), "beta": (0.29, 0.07), "label": "12+log10(N/H)"},
+    "FeH": {"alpha": (-0.69, 0.02), "beta": (0.17, 0.22), "label": "[Fe/H]"},
+}
+
+
+def fitcurve(M, alpha, beta):
+    return alpha + beta * (np.log10(M / unyt.Msun) - 10.0)
+
+
+for model in models:
+    label = models[model]["label"]
+
+    name = f"Stellar mass - Gas phase metallicity relation ({label})"
+    output_filename = f"Strom2022_{model}.hdf5"
+
+    alpha, alpha_err = models[model]["alpha"]
+    beta, beta_err = models[model]["beta"]
+
+    Z = fitcurve(M_star, alpha, beta)
+    Zpp = fitcurve(M_star, alpha + alpha_err, beta + beta_err)
+    Zpm = fitcurve(M_star, alpha + alpha_err, beta - beta_err)
+    Zmp = fitcurve(M_star, alpha - alpha_err, beta + beta_err)
+    Zmm = fitcurve(M_star, alpha - alpha_err, beta - beta_err)
+    Z_max = np.maximum(np.maximum(Zpp, Zpm), np.maximum(Zmp, Zmm))
+    Z_min = np.minimum(np.minimum(Zpp, Zpm), np.minimum(Zmp, Zmm))
+
+    Z_scatter = unyt.unyt_array([Z - Z_min, Z_max - Z], units="dimensionless")
+    Z = unyt.unyt_array(Z, units="dimensionless")
+
+    # Write everything
+    processed = ObservationalData()
+    processed.associate_x(
+        M_star, scatter=None, comoving=False, description="Galaxy Stellar Mass"
+    )
+    processed.associate_y(
+        Z,
+        scatter=Z_scatter,
+        comoving=False,
+        description=f"Gas phase metallicity ({label})",
```

Comment:
See my comment above
```suggestion
        description=f"Metallicity ({label})",
```